microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -386,10 +386,14 @@
     };
 
     function getMicrobiomeMapsRasterizedPlots(state) {
-        if(state.dataset_type.value === 'WGS') {
-            var pcoa_plots = ['tmi_WGS_gut_multipop', 'tmi_WGS_gut_lifestage'];
+        if(state.dataset_site.value === 'gut') {
+            if(state.dataset_type.value === 'WGS') {
+                var pcoa_plots = ['tmi_WGS_gut_multipop', 'tmi_WGS_gut_lifestage'];
+            } else {
+                var pcoa_plots = ['tmi_16S_gut_all_samples', 'tmi_16S_gut_multipop', 'tmi_16S_gut_lifestage', 'tmi_16S_gut_builtenv'];
+            }
         } else {
-            var pcoa_plots = ['tmi_16S_gut_all_samples', 'tmi_16S_gut_multipop', 'tmi_16S_gut_lifestage', 'tmi_16S_gut_builtenv'];
+            var pcoa_plots = [];
```

Comment:
So non-gut datasets don't get PCoA plots?

Reply:
Correct. For the time being, skin is the only non-gut sample type using this report template and it was decided that not showing any PCoA plots was preferable to investing the time in experimenting with various options to display. That decision might be re-evaluated down the road.